diff --git a/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js b/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js
index 6f6a928..41b2dc5 100644
--- a/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js
+++ b/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js
@@ -30,10 +30,10 @@ export const useActiveHeaderLinks = ({ headerLinkSelector, headerAnchorSelector,
             // notice the `scrollTop` might not be exactly equal to `offsetTop` after clicking the anchor
             // so we add offset
             // if has scrolled past this anchor
-            const hasPassedCurrentAnchor = scrollTop >= ((_b = (_a = anchor.parentElement) === null || _a === void 0 ? void 0 : _a.offsetTop) !== null && _b !== void 0 ? _b : 0) - offset;
+            const hasPassedCurrentAnchor = scrollTop >= ((_b = (_a = anchor.parentElement.parentElement) === null || _a === void 0 ? void 0 : _a.offsetTop) !== null && _b !== void 0 ? _b : 0) - offset;
             // if has not scrolled past next anchor
             const hasNotPassedNextAnchor = !nextAnchor ||
-                scrollTop < ((_d = (_c = nextAnchor.parentElement) === null || _c === void 0 ? void 0 : _c.offsetTop) !== null && _d !== void 0 ? _d : 0) - offset;
+                scrollTop < ((_d = (_c = nextAnchor.parentElement.parentElement) === null || _c === void 0 ? void 0 : _c.offsetTop) !== null && _d !== void 0 ? _d : 0) - offset;
             // if this anchor is the active anchor
             const isActive = isTheFirstAnchorActive ||
                 (hasPassedCurrentAnchor && hasNotPassedNextAnchor);
